src/averageAges.js

```diff
@@ -14,8 +14,19 @@
  *
  * @return {number}
  */
-function calculateMenAverageAge(people, century) {
-  // write code here
+function averager(people) {
+  const agesOnly = people.map((person) => person.died - person.born);
+
```

Comment:
You can calculate age inside reduce, it will help you to make amount of iterating less